.argo-ci/ci.yaml

```diff
@@ -44,7 +44,8 @@ spec:
           repo: "{{workflow.parameters.repo}}"
           revision: "{{workflow.parameters.revision}}"
     container:
-      image: argoproj/argo-cd-ci-builder:latest
+      image: argoproj/argo-cd-ci-builder:v0.12.0
```

Comment:
Ah yes. Because I am changing the meaning of `kustomize` from 1 to 2, the unit tests will fail unless I publish a new "builder image". But since the same build image is used for other branches (e.g. release-0.11) it means if everyone is using `:latest`, then I will break the release-0.11 branch builds. It was an eventuality that we would need to start versioning the builder images, this was the first time we needed to.